plugins/sqlfluff-templater-dbt/test/templater_test.py

```diff
@@ -70,10 +72,24 @@ def test__templater_dbt_templating_result(
         fname=os.path.join(project_dir, "models/my_new_project/", fname),
         config=FluffConfig(configs=DBT_FLUFF_CONFIG),
     )
-    assert (
-        str(templated_file)
-        == open("plugins/sqlfluff-templater-dbt/test/fixtures/dbt/" + fname).read()
+    template_output_folder_path = (
```

Comment:
This code is pretty complex -- I have a hard time following it. Could we make the additions clearer, perhaps with some comments?